client/signup/steps/user/index.jsx

```diff
@@ -124,7 +124,7 @@ export class UserStep extends Component {
 			}
 		} else if ( 1 === getFlowSteps( flowName ).length ) {
 			// Displays specific sub header if users only want to create an account, without a site
-			subHeaderText = translate( 'Welcome to the wonderful WordPress.com community' );
+			subHeaderText = translate( 'Welcome to the WordPress.com community.' );
```

Comment:
:disappointed: 23 existing translations will be lost with this change.

Reply:
Should we be concerned about changing these translations? /cc @Automattic/i18n

Reply:
The alliteration occurs in only in a few languages, and as far as we know it's only an issue in English. 

https://translate.wordpress.com/projects/wpcom/-all-translated/210474/

@akirk might give us an authoritative opinion on _**bei der wunderbaren WordPress.com-Community**_ assuming German-speakers pronounce it like _VordPress.com_, which I think they don't :)

My guess is that we could update the original string to `Welcome to the WordPress.com community.` and keep the translations as they are, unless you'd like to remove 'wonderful' in all translations. In the latter case, it might be best to have them re-translated just in case the adjective modifies the noun in some way in `x` language.

Reply:
Given that we'll probably have substantial copy changes with the new personality Signup steps, I'd also vote for holding off on the new non-EN translations. @akirk @ramonjd do we need to do anything in particular to have those ignored?

Reply:
Our awesome translators have started working on the new string: https://translate.wordpress.com/projects/wpcom/-all-translated/244254/

It might be useful just to have this string retranslated. 'Wonderful', putting aside any phonetic inconveniences, does alter the tone of the sentence.